storage/markdown/docs_ag2_ai_latest_docs_user-guide_models_anthropic.md

```
@@ -167,10 +167,10 @@ If you have been using `autogen` or `pyautogen`, all you need to do is upgrade i
 or 
 
 ```
-`[](#__codelineno-2-1)pipinstall-Upyautogen[anthropic] `
+`[](#__codelineno-2-1)pipinstall-Uag2[anthropic] `
 ```
 
-as `pyautogen`, `autogen`, and `ag2` are aliases for the same PyPI package. 
+as `ag2`, `autogen`, and `ag2` are aliases for the same PyPI package. 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)